composer.json

```diff
@@ -7,7 +7,7 @@
         "php": "^8.2"
     },
     "require-dev": {
-        "laravel/framework": "^12.35"
+        "laravel/framework": "^12.40"
```

Comment:
Was this meant to be `^12.39`? 👀 Or is this in preparation for a release tomorrow?

Assuming it's prepared for a release. 👍🏻 I'll release this once `12.40` comes out. 👍🏻